src/modules/menu/components/menu/reportProblem/ReportProblemButton.vue

```diff
@@ -0,0 +1,222 @@
+<script setup>
+import { computed, nextTick, ref } from 'vue'
+
+import sendFeedback from '@/api/feedback.api'
+import { createShortLink } from '@/api/shortlink.api'
+import ImportFileLocal from '@/modules/menu/components/advancedTools/ImportFile/ImportFileLocal.vue'
+import HeaderLink from '@/modules/menu/components/header/HeaderLink.vue'
+import ModalWithBackdrop from '@/utils/components/ModalWithBackdrop.vue'
+import log from '@/utils/logging'
+
+// comes from https://v2.vuejs.org/v2/cookbook/form-validation.html#Using-Custom-Validation
+const EMAIL_REGEX =
+    /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
```

Comment:
Instead of copying this regex from FeedbackButton, better create a util function in src/utils/utils.js `isEmailValid()` so it can be reused in both formular.